Update tests with sirepo-bluesky release

jennmald opened the pull request.

jklynch commented on deep_beamline_simulation/tests/test_ophyd_attempt.py:
```diff
@@ -0,0 +1,68 @@
+               'userName': None,
+               'visibleMethods': []},
+               'state': 'ok'}
+    assert str(expected['state']) == str(response_auth_guest_login.json()['state'])
```
Can tests like these be dictionary vs dictionary rather than str vs str?

jklynch commented on deep_beamline_simulation/tests/test_ophyd_attempt.py:
```diff
@@ -0,0 +1,68 @@
+    session = requests.Session()
+    response_sim_list = session.post("http://localhost:8000/simulation-list", json={"simulationType": "srw"})
+    response_auth_guest_login = session.post("http://localhost:8000/auth-guest-login/srw")
+    files = {"file": open("../../example-2.zip", "rb"), "folder": (None, "/foo")}
```
use deep_beamline_simulation.__path__ to figure out the absolute path to example-2.zip

jklynch replied:
that reminds me "example-2.zip" needs a more informative name

jklynch commented on tensorfile.py:
```diff
@@ -10,8 +9,8 @@
-file_name2 = 'accuracy.txt'
-file2 = open(file_name2, 'r')
+file_name2 = "accuracy.txt"
+file2 = open(file_name2, "r")
```
I recommend using the `while` statement whenever possible for opening and closing files to save yourself the trouble of 1) remembering to close the file when nothing goes wrong (which you have done) and 2) remembering to close the file when something goes wrong.

mrakitin replied:
`while` -> `with`?

jklynch replied:
Yes, my mistake. I meant ``with``.

jklynch reviewed (changes requested):
did accuracy.txt and loss.txt get deleted?

jennmald commented:
> did accuracy.txt and loss.txt get deleted?

I updated them when making the presentation and had yet to push the changes since I ran it so many times to get graphs. I updated them and put them into gitignore since I didn't want to update them each time I ran the tensor file.

mrakitin commented on tensorfile.py:
```diff
@@ -1,20 +1,16 @@
-file1 = open(file_name1, "r")
-for line in file1:
+with open("loss.txt", 'r') as file1:
+    line = file1.readline()
```
Did you intend to read only one first line? Or did you want to read all lines? For the latter case, `file1.read()` or `file1.readlines()` can be used.

mrakitin reviewed (commented):
@jennmald, I had some comments before this PR was merged. Please have a look at them as they may be useful for your work on tests (#29).

jennmald commented on deep_beamline_simulation/tests/test_ophyd_attempt.py:
```diff
@@ -0,0 +1,196 @@
+
+
+def test_runSimulation():
+    session = requests.Session()
+    response_sim_list = session.post(
+        "http://localhost:8000/simulation-list", json={"simulationType": "srw"}
+    )
+    response_auth_guest_login = session.post(
+        "http://localhost:8000/auth-guest-login/srw"
+    )
+    sirepo_simulations_dir = (
+        Path(deep_beamline_simulation.__path__[0]).parent / "sirepo_simulations"
+    )
+    simulation_zip_file_path = sirepo_simulations_dir / "sim_example.zip"
+    files = {
+        "file": open(str(simulation_zip_file_path), "rb"),
+        "folder": (None, "/foo"),
+    }
+    response_import_file = session.post(
+        "http://localhost:8000/import-file/srw", files=files
+    )
```
I decided to create a function to handle some of the repeated code. I will try to put all of these changes on the same PR (#29 )

jennmald commented on deep_beamline_simulation/tests/test_ophyd_attempt.py:
```diff
@@ -0,0 +1,196 @@
+    )
+    for i in range(1000):
+        state = (response_run_simulation.json())["state"]
+        if state == "completed" or state == "error":
+            state = (response_run_simulation.json())["state"]
+        if state == "completed" or state == "error":
```
fixed!

jennmald commented on deep_beamline_simulation/tests/test_ophyd_attempt.py:
```diff
@@ -0,0 +1,196 @@
+        "y_units": "ph/s/.1%bw/mm^2",
+        "z_range": [210561.34375, 1.8392213897609216e16],
+    }
+    actual_summary_data = response_run_simulation.json()["summaryData"]
```
fixed this as well, I had the assert at first then was second guessing what I was testing in it. The values stay the same for the simulation.
